src/components/NotePopup/NotePopup.js

```diff
@@ -89,6 +120,7 @@ const mapStateToProps = state => ({
   isDisabled: selectors.isElementDisabled(state, 'notePopup'),
   isEditDisabled: selectors.isElementDisabled(state, 'notePopupEdit'),
   isDeleteDisabled: selectors.isElementDisabled(state, 'notePopupDelete'),
+  isStateDisabled: selectors.isElementDisabled(state, 'notePopopState'),
```

Comment:
I think it should be `notePopupState` instead?

Reply:
Resolved in dcb321c. Thank you!